WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -112,6 +112,8 @@ public WalletViewModel(UiContext uiContext, IWalletModel walletModel, Wallet wal
 
 		WalletCoinsCommand = ReactiveCommand.Create(() => Navigate(NavigationTarget.DialogScreen).To().WalletCoins(WalletModel));
 
+		WalletExcludedCoinsCommand = ReactiveCommand.Create(() => Navigate(NavigationTarget.DialogScreen).To().ExcludedCoins(WalletModel));
```

Comment:
I think you could use the command that is in `CoinJoinStateViewModel` to avoid duplication.